packages/nouns-contracts/contracts/governance/NounsDAOProposals.sol

```diff
@@ -539,21 +537,12 @@ library NounsDAOProposals {
         }
 
         NounsDAOTypes.Proposal storage proposal = ds._proposals[proposalId];
-        address proposer = proposal.proposer;
-        NounsTokenLike nouns = ds.nouns;
-
-        uint256 votes = nouns.getPriorVotes(proposer, block.number - 1);
-        bool msgSenderIsProposer = proposer == msg.sender;
+        bool msgSenderIsProposerOrSigner = proposal.proposer == msg.sender;
         address[] memory signers = proposal.signers;
```

Comment:
gas: we can do less checks once msg.sender matches the `proposer` or a `signer`, no need to do more checks.

Reply:
yes, fixed